lib/tapioca/helpers/rbi_files_helper.rb

```diff
@@ -161,7 +161,8 @@ def parse_and_index_files(index, files, number_of_workers:)
         RBI::Parser.parse_file(file)
       rescue RBI::ParseError => e
         say_error("\nWarning: #{e} (#{e.location})", :yellow)
-      end
+        next
```

Comment:
Typically, I use `next` for early "return" style of control flow when we have to skip instruction below it. If it's the last statement for the block, I'll generally use `nil`. Both indeed result in the same thing, I just find odd to have `next` be the last statement in the block since it's not skipping over any code, you're basically just using it to return `nil`.

It's just a nitpick though, I don't think there's any style guidelines related to this.